### ISSUE TITLE
Include release notes in releases on GitHub

### ISSUE THREAD

**Issue by GeorgLink** [chaoss/grimoirelab#218]
When I look at the releases on GitHub, there is no indication of what is new or changed.

It would be great to have a list of new features, bug fixes, and other changes in a simple list format. 

I know the information is available in https://github.com/chaoss/grimoirelab/blob/master/releases/NEWS
But it would be great to have a copy of that information in https://github.com/chaoss/grimoirelab/releases

**Comment by valeriocos** [comment]
@sduenas has an idea about how to improve the release notes. @sduenas please share your thoughts.

**Comment by sduenas** [comment]
Starting in October we're going to work in a different way to generate the news documents and the way we release the software.

Currently, NEWS document is not useful for anyone because it contains the commit logs which usually are very technical and they don't give enough context. It would be better to have something like in Perceval (see https://github.com/chaoss/grimoirelab-perceval/releases/tag/0.10.0). The problem is this releases are done manually.

My idea is to create these documents automatically with the software packages too. We'll start with this in Perceval and Arthur. Once we see it works, we will use it in all the packages and in GrimoireLab main repository too.

I'll keep you posted.

**Comment by GeorgLink** [comment]
Excellent. That would be great.

There is one project where I always enjoy reading the release notes because
they are funny: https://www.youneedabudget.com/release-notes/

**Comment by jgbarah** [comment]
Great! Please, keep mi in the loop!

**Comment by GeorgLink** [comment]
Also, the release notes don't have to be fancy. The commit logs would already be helpful to get started.

**Comment by vchrombie** [comment]
A small update on this.

We have been working on automating the release workflow using poetry, github actions, and [Bitergia/release-tools](https://github.com/Bitergia/release-tools). This issue will be addressed in this workflow.

Whenever the maintainer releases a new version, the github actions would create a new release on GitHub. We have tested it on the release-tools repository.

Example
https://github.com/Bitergia/release-tools/releases/tag/0.3.0
https://github.com/Bitergia/release-tools/blob/master/releases/0.3.0.md

Workflow file (in case if anyone wants to have a look)
https://github.com/Bitergia/release-tools/blob/master/.github/workflows/release.yml

We are slowly working on integrating these actions to all the tools on the GrimoireLab toolset. Once we set up everything, the releases will be generated automatically for every tool. :rocket:

**Comment by vchrombie** [comment]
Hi everyone, thanks for waiting on this issue patiently. We were working on integrating the release-tools with the GrimoireLab projects and we are close to completion.

Most of the release process is now automated and the release notes would be published on the GitHub as the part of the workflow. This is for the individual components and for grimoirelab repository.

I would like to thank @sduenas and @jjmerchante for pulling up all the efforts to make this happen.

This would help us define and understand the GrimoireLab releases more clearly.

Release notes for GrimoireLab: https://github.com/chaoss/grimoirelab/releases
Release notes for Perceval: https://github.com/chaoss/grimoirelab-perceval/releases

cc @GeorgLink @jgbarah

**Comment by GeorgLink** [comment]
Yay! Thanks for the progress report.

**Comment by sduenas** [comment]
This is already done. Our releases include now the full list of changes. For example: https://github.com/chaoss/grimoirelab/releases/tag/0.15.0
